contracts/token/library.cairo

```diff
@@ -85,9 +82,11 @@ namespace ERC1155:
     # Getters
     #
 
-    func uri{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (uri : felt):
-        let (uri) = ERC1155_uri.read()
-        return (uri)
+    func token_uri{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        token_id: Uint256
+    ) -> (token_uri : felt):
+        let (token_uri) = ERC1155_token_uri.read(token_id)
```

Comment:
No need to change anything, just wanted to mention - since the return names of the storage var and the func are equal, you can just `return ERC1155_token_uri.read(token_id)`.